test/integration/integration.ts

```diff
@@ -1,124 +1,187 @@
+/* eslint-disable no-console -- Console comments make it easier to debug when tests fail */
 import assert from "assert";
 
 import _ from "lodash";
 import { isValidXAddress } from "ripple-address-codec";
 
-import { Client } from "xrpl-local";
-import { errors } from "xrpl-local/common";
+import { FormattedOrderSpecification } from "xrpl-local/common/types/objects";
 import { isValidSecret } from "xrpl-local/utils";
 
+import {
+  Client,
+  LedgerResponse,
+  Prepare,
+  SubmitResponse,
+  TxResponse,
+} from "../../src";
+import { AccountOffer } from "../../src/common/types/commands";
+import {
+  FormattedTrustline,
+  SignedTransaction,
+} from "../../src/common/types/objects";
+import { GetBalances } from "../../src/ledger/balances";
+import { Transaction } from "../../src/models/transactions";
 import { generateXAddress } from "../../src/utils/generateAddress";
 import requests from "../fixtures/requests";
 
 import { payTo, ledgerAccept } from "./utils";
-import wallet from "./wallet";
+import { walletAddress, walletSecret } from "./wallet";
 
 // how long before each test case times out
 const TIMEOUT = 20000;
-const INTERVAL = 1000; // how long to wait between checks for validated ledger
 
+// eslint-disable-next-line node/no-process-env -- Allows the user to pass in different IP's for local rippled
 const HOST = process.env.HOST ?? "0.0.0.0";
+// eslint-disable-next-line node/no-process-env -- Allows the user to pass in different ports for local rippled
 const PORT = process.env.PORT ?? "6006";
 const serverUrl = `ws://${HOST}:${PORT}`;
 
 console.log(serverUrl);
 
-function acceptLedger(client) {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any -- The return type is legitimately any
+async function acceptLedger(client: Client): Promise<any> {
   return client.connection.request({ command: "ledger_accept" });
 }
 
-function verifyTransaction(testcase, hash, type, options, txData, account) {
+// eslint-disable-next-line max-params -- Required for running checks while we move through the test
+async function verifyTransaction(
```

Comment:
All of these helper methods should move to the `utils` file. These methods (plus the methods in the `utils` file) might need to be split between a couple of files in order to be easier to parse.